policy/transact/transact.go

```diff
@@ -0,0 +1,85 @@
+package transact
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/atomist-skills/go-skill"
+	"github.com/atomist-skills/go-skill/policy/goals"
+	"github.com/atomist-skills/go-skill/policy/storage"
+)
+
+type PreviousResult struct {
+	StorageId  string
+	ConfigHash string
+}
+
+func TransactPolicyResult(
+	ctx context.Context,
+	evalCtx goals.GoalEvaluationContext,
+	configuration skill.Configuration,
+	digest string,
+	previousResult *PreviousResult,
+	evaluationTs time.Time,
+	goalResults []goals.GoalEvaluationQueryResult,
```

Comment:
Perhaps we could group the policy-related ones into a struct? This method seems to have an excessive amount of parameters 😅

Reply:
I think it's fine for now. This isn't new. 😄